handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Restart services
```

Comment:
Running this handler is only necessary when changing the systemd service file.
I suggest renaming this handler to `Reload systemd configuration` and calling it only on that occasion. Ansible's `notify` may also be a list in order to call multiple handlers.

Also, `Actually restart services` is never directly invoced, but every time `Restart services` runs, so you could just add `    daemon_reload: True` to the `Actually restart services` handler and achieve the same outcome as now.

Reply:
That would reload the daemon multiple times.

Reply:
Done